suncityla/app/globals.css

```diff
@@ -3,6 +3,12 @@
 @tailwind utilities;
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700&display=swap');
 
+@layer utilities{
```

Comment:
Can we apply the styles directly to the components that need them instead of here in this global file?

Reply:
Sure!